libethash-cl/CLMiner.cpp

```diff
@@ -266,6 +266,17 @@ CLMiner::~CLMiner()
     kick_miner();
 }
 
+typedef struct {
+	uint32_t gid;
+	uint32_t mix[8];
+	uint32_t pad[7];
+} SearchResult;
+
+typedef struct {
+	SearchResult rslt[c_maxSearchResults];
+	uint32_t count;
```

Comment:
I don't fully understand why having the `count` in a struct with results make sense.

Reply:
Bot rslt and count are updated by search funtion to report a solution, so they're both included n the results struct.